src/Microsoft.DotNet.Build.Tasks/Targets/PerfTesting.targets

```diff
@@ -40,7 +39,9 @@
 
   <!-- Executable properties -->
   <PropertyGroup>
-    <XunitRunnerArgs>$(TargetFileName) -runnerhost $(TestHost) -runner $(XunitExecutable) -runid $(PerfRunId) -verbose</XunitRunnerArgs>
+    <PerfRunId Condition="'$(PerfRunId)' == ''">$(TargetFileName)-WindowsCore</PerfRunId>
```

Comment:
I can't imagine it would be reusable, especially since the unix runner is dnx based now